### ISSUE TITLE
Application not working anymore?

### ISSUE THREAD

**Issue by mxro** [MooseSaeed/wordleup#1]
When opening the URL http://wordleit.herokuapp.com/, I get

![image](https://github.com/MooseSaeed/wordleup/assets/1448524/133312f1-03be-4b41-a13f-050c8f135c27)

**Comment by MooseSaeed** [comment]
Yeah, the project isn't live anymore. I guess I'm going to archive the repo.

**Comment by MooseSaeed** [comment]
@mxro Yeah, the project isn't live anymore. I guess I'm going to archive it soon if I didn't have time to find an alternative for Heroku.
